src/linear_algebra/finite_dimensional.lean

```diff
@@ -283,6 +283,17 @@ let ⟨B, hB, B_fin⟩ := exists_is_basis_finite K V, ⟨g, hg⟩ := finite_dime
 
 variables {K V}
 
+/-- A module with dimension 1 has a basis of the form `{v}` for some `v : V`. -/
+lemma exists_is_basis_singleton (h : finrank K V = 1) :
```

Comment:
I would have defined this in two steps: derive `unique ι` from `finrank K V = 1` and `is_basis K (b : ι → V)`, and then use a lemma of the form
```lean
example {α : Type*} (s : set α) (hs : unique s) : ∃ x, s = {x} := by library_search
```
to conclude `exists_is_basis_singleton`. But the `library_search` above doesn't give any results, so I'm okay with keeping the lemma as-is (and I'll change it around when I update the bundled-basis branch).

Reply:
Okay, thank you! :-)